### ISSUE TITLE
Keep getting notifications that Proxyman regularly tries to add itself to background items, even though I disabled that.

### ISSUE THREAD

**Issue by huyz** [ProxymanApp/Proxyman#1762]
## Description

I went to Login Items and purposefully disabled Proxyman from background apps because I have only want it to run when I need it.
The problem is every few hours, Proxyman tries to re-add itself and I get notified.  I don't know which process is doing that.  None of my hundreds of other apps triggers this notification all the time.

![screenshot 2023-09-06T094821Z](https://github.com/ProxymanApp/Proxyman/assets/128394/e4eb6318-f1ee-4f58-8673-c758553f85c3)

## Steps to Reproduce

1. Install Proxyman
2. Go to Settings > Login Items and disable Proxyman.app from `Allow in the Background`

## Current Behavior

1. Every few hours you get a notification as screenshotted above

## Expected Behavior

1. No notifications

## Environment

- App version: Proxyman 4.10.0
- macOS version: macOS 13.5.1 (Ventura, ARM)

**Comment by NghiaTranUIT** [comment]
Hi @huyz, the macOS uses the term "Login Items" is incorrect. It's a [Helper Tool](https://docs.proxyman.io/basic-features/proxy-setting-tool) that Proxyman requires to override/revert the system HTTP Proxy when the app is launched or closed. It's also useful to gracefully revert the Proxy if the app is unexpectedly crashed.

The helper tool doesn't start in the Login time, it only needs when Proxyman is running.

If you want to get rid of the notification, You can uninstall the Helper Tool in Proxyman Menu -> Setting -> Advance Tab -> Uninstall Helper Tool.

**Comment by huyz** [comment]
It sounds like a useful tool. So I don't want to uninstall it.
It's just that I want the option to only run the tool when Proxyman is launched. And then it can stay running in the background until the end of login session. Isn't that possible?

**Comment by NghiaTranUIT** [comment]
It's a default behavior of the Helper Tool. When you enable it in the Setting, it works as you describe.

Proxyman Helper Tool does nothing if the main app is not running.

**Comment by huyz** [comment]
I think there's some misunderstanding.

It does do something when the main app is not running: it's triggering a notification.

I rebooted, never started Proxyman, and yet I get notified as above.

No other app does this.

**Comment by NghiaTranUIT** [comment]
It's odd. Let me investigate it 👍

**Comment by huyz** [comment]
The issue is gone.
And now I see VMware doing the same thing to me.
So this is probably a macOS issue actually (similar to how macOS often loses Privacy/Accessibility settings all the time).
Feel free to close this issue.

**Comment by NghiaTranUIT** [comment]
yes, properly a macOS issue and I'm not sure how to completely fix it from our end.
